smdebug/tensorflow/keras.py

```diff
@@ -122,9 +124,12 @@ def __init__(
         self.has_logged_unsupported_tensors_in_non_eager_execution = False
         self.prepared_tf2_collections = False
         self.prepared_gradient_tape_collections = False
+        # this flag indicates whether profiling is enabled for native TF2 training
+        self.is_profiler_enabled_for_native_training = False
+        self.closed = False
```

Comment:
Rename `self.closed` to something more descriptive.

Reply:
Done